src/dnx.coreclr/dnx.coreclr.cpp

```diff
@@ -397,126 +230,126 @@ extern "C" HRESULT __stdcall CallApplicationMain(PCALL_APPLICATION_MAIN_DATA dat
         L"AppDomainCompatSwitch",
     };
 
-    cchTrustedPlatformAssemblies = TRUSTED_PLATFORM_ASSEMBLIES_STRING_BUFFER_SIZE_CCH;
-    pwszTrustedPlatformAssemblies = (LPWSTR)calloc(cchTrustedPlatformAssemblies+1, sizeof(WCHAR));
-    if (pwszTrustedPlatformAssemblies == NULL)
-    {
-        goto Finished;
-    }
-    pwszTrustedPlatformAssemblies[0] = L'\0';
+    std::wstring trusted_platform_assemblies;
+    trusted_platform_assemblies.reserve();
```

Comment:
I was looking for this line at the very moment to add a note. Originally we allocated 64KB but I found that typically all the paths are ~8K so I removed `-#define TRUSTED_PLATFORM_ASSEMBLIES_STRING_BUFFER_SIZE_CCH (63 * 1024) //32K WCHARs` but realized that I did not remember setting the size here. Certainly I will fix that.